Unosquare.CodeChallenge/WarehouseTestingUnit/WarehouseServices/WarehouseServices_unitTest.cs

```diff
@@ -0,0 +1,145 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tests;
+using WarehouseCoreLib.DataAccess;
+using WarehouseModels.Models;
+using WarehouseServices.Services;
+using Assert = NUnit.Framework.Assert;
+using Theory = NUnit.Framework.TheoryAttribute;
+
+namespace WarehouseTestingUnit.WarehouseServices
+{
+    public class WarehouseServices_unitTest
+    {
+
+        [Fact]
+        public void Test_getAllCompanies_getAllCompaniesItems()
+        {
+            CompaniesServices service = new CompaniesServices(WarehouseHelper.createDBContext());
```

Comment:
This warehouse helper you should look into Fixtures and InMemoryDatabase with EF Core and use that instead

Reply:
on the commit 846bc3793eab7837989143e1bd91bd78e0bad180 , I create the fixtures and add the in memory the database